specs/corla.pvs

```diff
@@ -0,0 +1,923 @@
+%%% model of the Colorado Risk Limiting Audit (CORLA) tool.
```

Comment:
@dmzimmerman, let's make new branches and PRs, off `master`, for updates (as much as makes sense). it's a _lot_ easier to manage smaller, short-lived PRs.